### ISSUE TITLE
[FaloopIntegration] Empty message into 'System.ArgumentNullException: Value cannot be null.'

### ISSUE THREAD

**Issue by drejjmit** [SlashNephy/Divination#655]
I noticed a lot of empty announcements. If you look at their Discord, there are no coords until in this case 10 seconds. Could you ignore printing of spawns with no spawn data?

![image](https://github.com/SlashNephy/Divination/assets/15526476/d2f13c90-2bc1-4b41-9390-bfab213580a8)

```
10:30:23.578 | DBG | [FaloopIntegration] Event "message" = "[{\"type\":\"mob\",\"subType\":\"report\",\"data\":{\"action\":\"spawn\",\"id\":{\"mobId\":\"gunitt\",\"worldId\":\"omega\"},\"data\":{\"zoneId2\":\"the_tempest\",\"location\":null,\"timestamp\":\"2024-04-24T08:30:23.469Z\",\"window\":1,\"stage\":null}}}]"
10:30:23.578 | DBG | [FaloopIntegration] OnMobReport: unknown zone poi id found: 0
```

And the second announce 10 seconds later with the actual spawn data got an error so it was never printed in chat: 

```
10:30:33.876 | DBG | [FaloopIntegration] Event "message" = "[{\"type\":\"mob\",\"subType\":\"report\",\"data\":{\"action\":\"spawn_location\",\"id\":{\"mobId\":\"gunitt\",\"worldId\":\"omega\"},\"data\":{\"zonePoiId\":1034}}}]"
10:30:33.877 | ERR | [FaloopIntegration] HandleOnMessage
	System.ArgumentNullException: Value cannot be null. (Parameter 'key')
	   at System.Collections.Generic.Dictionary`2.FindValue(TKey key)
	   at System.Collections.Generic.Dictionary`2.TryGetValue(TKey key, TValue& value)
	   at Divination.FaloopIntegration.FaloopIntegration.OnMobReport(MobReportData data) in D:\a\Divination\Divination\FaloopIntegration\FaloopIntegration.cs:line 131
	   at Divination.FaloopIntegration.Faloop.FaloopSocketIOClient.HandleOnMessage(SocketIOResponse response) in D:\a\Divination\Divination\FaloopIntegration\Faloop\FaloopSocketIOClient.cs:line 171
```

I'm on **v6.58.9.0**

**Edit:**
Another example
```
13:17:08.121 | DBG | [FaloopIntegration] Event "message" = "[{\"type\":\"mob\",\"subType\":\"report\",\"data\":{\"action\":\"spawn\",\"id\":{\"mobId\":\"nunyunuwi\",\"worldId\":\"spriggan\"},\"data\":{\"zoneId2\":\"southern_thanalan\",\"location\":null,\"timestamp\":\"2024-04-26T11:17:07.948Z\",\"window\":1,\"stage\":null}}}]"
13:17:08.121 | DBG | [FaloopIntegration] OnMobReport: unknown zone poi id found: 0
13:17:11.879 | DBG | [FaloopIntegration] Ping
13:17:11.879 | DBG | [FaloopIntegration] Pong: 00:00:00.0000256
13:17:13.871 | DBG | [FaloopIntegration] Event "message" = "[{\"type\":\"mob\",\"subType\":\"report\",\"data\":{\"action\":\"spawn_location\",\"id\":{\"mobId\":\"nunyunuwi\",\"worldId\":\"spriggan\"},\"data\":{\"zonePoiId\":283}}}]"
13:17:13.871 | ERR | [FaloopIntegration] HandleOnMessage
	System.ArgumentNullException: Value cannot be null. (Parameter 'key')
	   at System.Collections.Generic.Dictionary`2.FindValue(TKey key)
	   at System.Collections.Generic.Dictionary`2.TryGetValue(TKey key, TValue& value)
	   at Divination.FaloopIntegration.FaloopIntegration.OnMobReport(MobReportData data) in D:\a\Divination\Divination\FaloopIntegration\FaloopIntegration.cs:line 131
	   at Divination.FaloopIntegration.Faloop.FaloopSocketIOClient.HandleOnMessage(SocketIOResponse response) in D:\a\Divination\Divination\FaloopIntegration\Faloop\FaloopSocketIOClient.cs:line 171
```
![image](https://github.com/SlashNephy/Divination/assets/15526476/f7575a29-ff63-4e20-a86a-3aa3165de0e0)

**Comment by SlashNephy** [comment]
Hi!
Sorry for the late response. I just released v6.58.10.
[Release 6.58.10 · SlashNephy/Divination](https://github.com/SlashNephy/Divination/releases/tag/6.58.10)

Can you close the Issue once you are sure the issue has been resolved?

**Comment by drejjmit** [comment]
I don't see the option in settings, but i'll just set the bool in the json for now

![image](https://github.com/SlashNephy/Divination/assets/15526476/2a8be45c-6db2-4c33-a8eb-2f30d3adb2e8)
![image](https://github.com/SlashNephy/Divination/assets/15526476/495befc0-1760-4768-aa8e-34e53b4dd618)

**Comment by SlashNephy** [comment]
Sorry. I just forgot to add checkbox...
https://github.com/SlashNephy/Divination/commit/52ac778e451b4b55845fd151580bb40bbdfa2176
Fixed in 6.58.11.
